src/server/src/main/docker/Dockerfile

```
@@ -38,6 +38,9 @@ ENV REAPER_SEGMENT_COUNT_PER_NODE=64 \
     REAPER_AUTO_SCHEDULING_PERIOD_BETWEEN_POLLS=PT10M \
     REAPER_AUTO_SCHEDULING_TIME_BEFORE_FIRST_SCHEDULE=PT5M \
     REAPER_AUTO_SCHEDULING_SCHEDULE_SPREAD_PERIOD=PT6H \
+    REAPER_AUTO_SCHEDULING_ADAPTIVE=false \
+    REAPER_AUTO_SCHEDULING_INCREMENTAL=true \
```

```suggestion
    REAPER_AUTO_SCHEDULING_ADAPTIVE=true \
    REAPER_AUTO_SCHEDULING_INCREMENTAL=false \
```

If you don't mind, I think we should set defaults to adaptive=true and incremental=false. Incremental repairs are only safe with Cassandra 4.0+, so I wouldn't want to set defaults that can create chaos.